content/source/docs/cloud/workspaces/vcs.html.md

```diff
@@ -0,0 +1,79 @@
+
+-> **API:** See the [Update a Workspace endpoint](../api/workspaces.html#update-a-workspace) (`PATCH /organizations/:organization_name/workspaces/:name`).
+
+To change the VCS settings for an existing workspace, choose "Version Control" from the workspace's "Settings" menu. The version control page includes the following settings:
+
+- Which VCS repository to use.
+- Which directories in the repository should trigger Terraform runs when they change.
+- Which branch of the repository to use.
+- Whether to download the repository's submodules.
+
+For most of the settings on this page, you must save any changes with the "Update VCS settings" button at the bottom of the page.
+
+## VCS Connection and Repository
+
+The first item in the version control settings lets you select a new VCS repository or disconnect from the current repository.
+
+-> **Note:** Depending on the current status, this control can appear as a "Connect to version control" button or a "Change VCS connection" link.
+
+Connecting or disconnecting a VCS repository occurs on a separate page, split into three screens:
+
+1. On the first screen, choose your VCS provider (or choose "No VCS connection" to disconnect the workspace from version control).
+
+    -> **Note:** If you haven't added a VCS provider for your organization yet, choosing one here will prompt you to configure it. See [Connecting VCS Providers](../vcs/index.html) for more information. Only organization owners can configure VCS providers.
+
+1. On the second screen, choose a repository from the filterable list. This screen is skipped if you chose "No VCS connection".
+
+    Some VCS providers limit the list's size. If a repository isn't listed, you can still choose it by name; scroll to the bottom of the list and enter its ID in the text field.
+
+    -> **Note:** For some VCS providers, this list includes a drop-down menu for changing which account's repositories are shown. Other providers combine all available accounts into a single list.
+
+1. On the third screen, confirm or cancel your choice. Confirmed changes will be saved immediately, and you will be returned to the "Version Control" settings page.
+
+See also:
+
+- [Creating Workspaces](./creating.html)
+- [Connecting VCS Providers to Terraform Cloud](../vcs/index.html)
+
+-> **API:** If you need to change VCS connections for many workspaces at once, consider automating the changes with the [Update a Workspace endpoint](../api/workspaces.html#update-a-workspace). This is most common when moving a VCS server, or when a vendor deprecates an older API version.
+
+## Automatic Run Triggering
+
+-> **Note:** This setting only affects workspaces that specify a Terraform working directory. [The working directory setting](./settings.html#terraform-working-directory) can be found on the "General" settings page.
```

Comment:
Maybe remove this note or combine it with paragraph 2? It feels a bit whiplash-y to think about workspaces that do, then workspaces that don't, and then workspaces that do again.

Reply:
Mm, yeah. Artifact of the previous layout of this info.